### PR TITLE
Link party

### PR DESCRIPTION
**Pull request by Stearnzy**
Made parties info movie title a link to show page
Made test and necessary VCRs :: passing

Rubocop fixes

### CODE REVIEW

**Comment by Stearnzy** [comment]
FINALLY

File: spec/features/dashboard/dashboard_spec.rb [hunk 1]
```diff
@@ -67,10 +72,23 @@
...
       expect(page).to have_content('December 1, 2020')
       expect(page).to have_content('2:00 PM')
       expect(page).to have_content('Invited')
     end
   end
+
+  it "Clicking on the movie name in the viewing party display takes me to the movie show page" do
+    Guest.create(party_id: @party.id, user_id: @current_user.id, attending: true)
+
+    VCR.use_cassette('movie_details') do
```
Comment (Eternal-Flame085) [inline spec/features/dashboard/dashboard_spec.rb:85]: You did nothing different from what you had. Travis is a DC fan

File: app/views/dashboard/index.html.erb [hunk 2]
```diff
@@ -23,7 +23,7 @@
   <h2>Viewing Parties</h2>
   <% current_user.parties.each do |party| %>
     <section id='party-<%= party.id %>'>
-      <%= party.movie.title %><br>
+      <%= link_to party.movie.title, "/movies/#{party.movie.api_id}" %> <br>
```
Comment (Eternal-Flame085) [inline app/views/dashboard/index.html.erb:26]: Nice addition

**Review by Eternal-Flame085 — APPROVED** [review]

**Review by stillsheryl — APPROVED** [review]
Great job team!!

**Pull request merged**